test/common/TestBed.hpp

```diff
@@ -134,10 +134,10 @@ namespace RcclUnitTesting
                                                   int const numGroupCalls);
 
     // Helper function that splits up GPUs to the given number of processes
-    static std::vector<std::vector<int>> GetDeviceIdsList(int const numProcesses,
+    std::vector<std::vector<int>> GetDeviceIdsList(int const numProcesses,
                                                           int const numGpus,
                                                           int const ranksPerGpu);
-    static std::vector<std::vector<int>> GetDeviceIdsList(int const numProcesses,
+    std::vector<std::vector<int>> GetDeviceIdsList(int const numProcesses,
                                                           int const numGpus);
```

Comment:
Maybe consider passing `gpuPriorityOrder` in as an argument rather than making these non-static.

Reply:
Thanks for the suggestion. I have incorporated it in my latest PR. https://github.com/ROCm/rccl/pull/1403